### ISSUE TITLE
Handling masked characters

### ISSUE THREAD

**Issue by tmpfs** [PierreMarchand20/asciinema_automation#15]
Hi, thanks for the library, it's really useful!

I think the code cannot handle masked characters (eg: password) input.

I am trying to automate a program that accepts passwords but echoes `*` for obvious reasons.

I don't know much python but I think you are always expecting the same (escaped for regex) character out here: https://github.com/PierreMarchand20/asciinema_automation/blob/main/asciinema_automation/instruction.py#L67-L68

Would it be easy to add something like:

```
#$ sendmaskline * my-mock-password
```

What do you think?

**Comment by PierreMarchand20** [comment]
Hello, could you try the `sendcharacter ` instruction? (see example [here](https://github.com/PierreMarchand20/asciinema_automation/blob/main/examples/backward_search.sh#L12)) there is no expected value here, so I guess it should work.

**Comment by PierreMarchand20** [comment]
ah I missed the fact that you want to echo * instead. I am not sure it is up to this package to decide what is echoed. It just sends characters and sometimes check that we get the expected output. If your command expects a password, it is up to the command to echo * instead when you write your password I guess.

**Comment by PierreMarchand20** [comment]
using this kind of shenanigans I guess: https://stackoverflow.com/questions/3980668/how-to-get-a-password-from-a-shell-script-without-echoing

**Comment by tmpfs** [comment]
Hey @PierreMarchand20, you were right the first time, I don't need to get the echoed value back. It's just your code hangs on those lines as what's echoed is not what's input. Sending a character does work but that's not really practical and I am running into other issues I need to figure out.

**Comment by PierreMarchand20** [comment]
I think sendcharacter should work with several characters actually. In which case, you could use `sendcharacter yourpassword`.

If this does not work, could you make a minimal working example I can work with?

**Comment by tmpfs** [comment]
Thanks for taking the time to look into this but this was urgent for me so I decided to roll my own version, I have called it [anticipate](https://github.com/tmpfs/anticipate/). It's a bit rough around the edges as I have only spent a day on it, but it works for the CLI demos I need to do right now.

The syntax is based on yours and I give you credit in the README.

The latest version of asciinema is in Rust too so I might see if I can get them to fold this in to main program.
